README.md

```diff
@@ -63,7 +63,7 @@ The package offers wide configurability using Options. You can easily override a
 ### Usage
 ```go
 authProvider := client.NewWithSecret("my-client-id", "my-client-secret")
-resp, err := authProvider.ManagedHTTPClient().Get("https://authservice.bitrise.io/token-endpoint")
+resp, err := authProvider.ManagedHTTPClient().Get("https://myservice.services.bitrise.io/")
```

Comment:
t was confusing, because this example shows how to get an external site with the token, NOT how to get a token